### PR TITLE
Support multiline pythran spec

### PR DESCRIPTION
**Pull request by serge-sans-paille**
This makes it possible, in case of very long signature, to still follow
pep8 recommendation.

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
@pbrunet : this one is ok for review!

File: pythran/tests/test_spec_parser.py [hunk 1]
```diff
@@ -50,3 +50,39 @@ def test_invalid_specs2(self):
...
+            '''
+        pythran.spec_parser(code)
+
+    def test_crappy_spec0(self):
+        code = '''
+#      pythran export foo(int) this is a pythran test
+def foo(i,j,k): return
+            '''
+        pythran.spec_parser(code)
+
```
Comment (pbrunet) [inline pythran/tests/test_spec_parser.py]: Do we need a test with a comment on a next line? (I guess it is the same as crappy export)

Comment (serge-sans-paille) [inline pythran/tests/test_spec_parser.py]: On Sat, Dec 05, 2015 at 10:41:09AM -0800, pbrunet wrote:
> Do we need a test with a comment on a next line? (I guess it is the same as crappy export)
Better safe than sorry, so let's add it :-)

**Comment by serge-sans-paille** [comment]
ready!

**Pull request merged**